src/backend/storeManagers/sideload/games.ts

```diff
@@ -71,7 +72,10 @@ export async function launch(
   /* eslint-disable-next-line @typescript-eslint/no-unused-vars */
   launchArguments?: LaunchOption
 ): Promise<boolean> {
-  return launchGame(appName, getGameInfo(appName), 'sideload')
+  // wait for the game to launch and finish running
+  return Promise.resolve(
+    await launchGame(appName, getGameInfo(appName), 'sideload')
+  )
```

Comment:
This still does not do anything. What are you actually trying to do/fix? As long as callers of this function `await` it, they'll already `await` the Promise returned by `launchGame`

Reply:
oh that's right actually, then why did this work in the first place lol